module/os/linux/zfs/vdev_disk.c

```diff
@@ -1042,3 +1042,6 @@ param_set_max_auto_ashift(const char *buf, zfs_kernel_param_t *kp)
 
 	return (0);
 }
+
+ZFS_MODULE_PARAM(zfs_vdev, zfs_vdev_, open_timeout_ms, UINT, ZMOD_RW,
```

Comment:
not sure if we want to document this in `man/man4/zfs.4`?

Reply:
Nice catch. We should document this in the man page.